index.html

```diff
@@ -25,7 +26,174 @@ <h1>If These Lands Could Talk</h1>
 
 		<!-- Footer -->
 
+		<footer>
+			<div class="wrapper">
+				<div class="footer-main">
+					<div class="footer-logo-btn">
+						<!-- logo -->
+						<a href="./index.html">
+							<img
+								src="./assets/icons/global/itlct-logo.svg"
+								alt="If These Lands Could Talk Logo"
+							/>
+						</a>
+
+						<!-- Button for modal -->
+						<button class="btn-la">Land Acknowledgement</button>
+					</div>
+					<!-- Links list -->
+					<!-- need to replace with the real page links -->
+					<ul>
+						<li>
+							<a href="./index.html">Home</a>
+						</li>
+						<li>
+							<a href="#">Our Impact</a>
+						</li>
+						<li>
+							<a href="#">Our Values</a>
+						</li>
+						<li>
+							<a href="#">Events & Programs</a>
+						</li>
+						<li>
+							<a href="#">Our Team</a>
+						</li>
+						<li>
+							<a href="#">Join the Team</a>
+						</li>
+						<li>
+							<a href="#">Contact</a>
+						</li>
+						<li>
+							<a href="#">Support Us</a>
+						</li>
+					</ul>
+
+					<!-- form -->
+					<form action="" class="">
+						<h4>Stay up to date with us</h4>
+						<p>
+							Register with our newsletter to receive updates
+							<br />
+							on our website, events, and more!
+						</p>
+						<div class="text-input">
+							<input
```

Comment:
I think this input field is supposed to be wider on desktop (400px). CSS width probably won't work. You'll want to use the `size` attribute. 

<img width="490" alt="Screen Shot 2024-01-11 at 11 57 00 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/86d8ea88-814c-4830-8093-246cdc64b557">